docs/modules/user-guide/partials/proc_tagging-and-pushing-a-registry-container-image.adoc

```diff
@@ -0,0 +1,19 @@
+[id="building-a-custom-devfile-registry_{context}"]
+= Tagging and pushing a registry container image
+
+[role="_abstract"]
+Use the devfile index container image to create a container in which you build, run, and deploy your application.
+
+.Procedure
+
+. Consult the documentation for your container registry and log into the documentation with the Docker CLI.
```

Comment:
"log into the documentation"? Was that a typo? Do we mean "log into the registry"?